tools/codeowners.bzl

```diff
@@ -10,22 +10,34 @@ def _codeowners_impl(ctx):
         fail("Either team or teams must be set.")
     if len(ctx.attr.team) > 0 and len(ctx.attr.teams) > 0:
         fail("Both team and teams can not be set at the same time.")
+    if len(ctx.attr.pattern) > 0 and len(ctx.attr.patterns) > 0:
+        fail("Both pattern and patterns can not be set at the same time.")
 
     teams = ctx.attr.teams
     if len(ctx.attr.team) > 0:
         teams = [ctx.attr.team]
 
+    # Default to empty pattern (matching path only)
+    patterns = [""]
```

Comment:
what about setting the default to `**` (everything in this package)?

Seems more bazel-idiomatic.

Reply:
Is the double-globbing syntax supported? I can't find any documentation of it.

From the [about page](https://help.github.com/en/articles/about-code-owners):

```codeowners
# In this example, @doctocat owns any files in the build/logs
# directory at the root of the repository and any of its
# subdirectories.
/build/logs/ @doctocat
```